net-libs/libwebsockets/libwebsockets-2.1.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
The copyright should be extended to 2017, please consider using `repoman commit` as much as possible.